website/hugo.yaml

```diff
@@ -19,7 +19,7 @@ relativeURL: true
 contentDir: "docs-pre-processed"
 archetypeDir: "archetypes"
 layoutDir: "layouts"
-publishDi: "public"
```

Comment:
wut

Reply:
`publishDir`?

Reply:
Diff is weird here ? I've fixed the typo

Reply:
just was curious why it is working on master then, that's fine